src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -20,7 +21,14 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD_RUNTIME</DefineConstants>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
Can't we place these in the common.props file?

Reply:
nuget doesn't put them there when doing a package update.  we can talk about moving them, but that means we have to do that every time we update our packages.

---
In reply to: [251941091](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/838#discussion_r251941091) [](ancestors = 251941091)